data_steward/cdr_cleaner/cleaning_rules/remove_ehr_data_without_consent.py

```diff
@@ -202,13 +223,13 @@ def setup_validation(self, client):
         """
         Run required steps for validation setup
         """
-        raise NotImplementedError("Please fix me.")
+        pass
```

Comment:
revert this.

Reply:
Sorry, I thought this change was brought up at one of our meetings